src/main/java/HandleTruth.java

```diff
@@ -16,10 +16,11 @@ public static void wordCount() {
             }
             wordMap.put(word, 1);
         }
+        /*
```

Comment:
if some codes are not needed any more, remove them instead of commenting it out.